beeline-core/src/main/java/io/honeycomb/beeline/tracing/propagation/BeelineServletFilter.java

```diff
@@ -87,9 +87,11 @@ protected BeelineServletFilter(final String serviceName,
                                    final List<String> excludePaths,
                                    final Function<HttpServletRequest, String> requestToRedispatchSpanName,
                                    final Function<io.honeycomb.beeline.tracing.propagation.HttpServerRequestAdapter, String> requestToSpanName,
-                                   final PathMatcher pathMatcher) {
+                                   final PathMatcher pathMatcher,
+                                   final PropagationCodec<Map<String, String>> PropagationCodec) {
```

Comment:
I'm having trouble parsing this -- is the variable name `PropagationCodec`? Isn't that the class name as well?

Reply:
Yeah, the parameter name should be `propagationCodec` (lowercase p) to differentiate the type from the parameter name. Java doesn't stop you reusing a class name as a property name like other languages do.